pkg/reconciler/autoscaling/kpa/kpa_test.go

```diff
@@ -1107,6 +1107,30 @@ func TestReconcile(t *testing.T) {
 				WithPAMetricsService(privateSvc), WithObservedGeneration(1),
 			),
 		}},
+	}, {
+		Name: "mark initial scale reached for an existing inactive PA",
+		Key:  key,
+		Ctx: context.WithValue(context.WithValue(context.Background(), asConfigKey{}, initialScaleZeroASConfig()), deciderKey{},
```

Comment:
do we need `initialScaleZeroASConfig` here? I'd presume we don't want that, since we're simulating pre 0.17

Reply:
yeah, we don't need it because we are simulating pre 0.17